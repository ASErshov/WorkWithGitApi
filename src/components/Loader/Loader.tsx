import React from 'react'
import styles from './Loader.module.css'

const Loader: React.FC =():React.ReactElement => {
    return (
        <div className={styles.loading}>
            <div>
            </div>
            <div>
            </div>
            <div>
            </div>
            <div>
            </div>
        </div>
    )
}

export default Loader