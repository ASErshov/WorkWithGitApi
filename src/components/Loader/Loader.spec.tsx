import React from 'react'
import renderer from 'react-test-renderer'
import { mount } from 'enzyme'

import styles from './Loader.module.css'

import Loader from './index'

describe('<Loader />', () => {
  const Component = (
    <Loader/>
  )

  it('Snapshot is correct', () => {
    const wrapper = renderer.create(Component).toJSON()

    expect(wrapper).toMatchSnapshot()
  })


})
