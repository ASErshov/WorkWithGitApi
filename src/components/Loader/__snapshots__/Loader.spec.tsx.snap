// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> Snapshot is correct 1`] = `
<div
  className="loading"
>
  <div />
  <div />
  <div />
  <div />
</div>
`;

exports[`<Loader /> Snapshot is correct 1`] = `
<div
  className="loading"
>
  <div />
  <div />
  <div />
  <div />
</div>
`;
